Scenes/Highscore/Highscore.gd

```diff
@@ -11,7 +11,40 @@ func _on_HTTPRequest_request_completed(result, response_code, headers, body):
 	var highscoretext = ""
 	var num = 1
 	for entry in json.dreamlo.leaderboard.entry:
-		highscoretext += "%3s: %40s %50s %20s\n" % [num, entry.name, entry.date, entry.score]
+
+		var date = entry.date
+		var splitup = date.split(" ", false, 0)
+		if splitup.size() > 3:
```

Comment:
What if size() < 3? Then you would later access elements in the array, that do not exist.
Please change to != 3

Reply:
```suggestion
		if splitup.size() != 3:
```